server/src/main/resources/org/eclipse/lsp/cobol/core/strategy/SpecialTokenHandling.properties

```diff
@@ -21,5 +21,17 @@ COMMENTTAG=
 COMMENTENTRYTAG=
 COMMENTENTRYLINE=
 COMMENTLINE=
+C_CHAR=C
```

Comment:
How about `AR`, `CO` and others?

Reply:
No need, this in only for tokens whose names don't match the assigned patterns